pkg/virt-controller/watch/snapshot.go

```diff
@@ -688,7 +688,7 @@ func (s *vmSnapshotSource) Lock() (bool, error) {
 	vmCopy.Status.SnapshotInProgress = &s.snapshot.Name
 	controller.AddFinalizer(vmCopy, sourceFinalizer)
 
-	_, err := s.client.VirtualMachine(vmCopy.Namespace).UpdateStatus(vmCopy)
+	_, err := s.client.VirtualMachine(vmCopy.Namespace).Update(vmCopy)
```

Comment:
the status will be ignored here during Update.

I think you'll have to call both update and updateStatus here.  Update if the metadata changes and updateStatus if the status changes.

Reply:
That is pretty bogus.

Reply:
you can disable the status subresource if you want. it had to be disabled for the vmi api for a similar reason.

Reply:
nah I think I can make it work, but it definitely completes things

Reply:
Updated